### DISCUSSION TITLE
Investigate increasing developer productivity with pull requests environments

### DISCUSSION THREAD

**Discussion by waveywaves** [getredash/redash#5915] [General]
I would like to make life easier for existing and new Redash contributors including maintainers by implementing [Uffizzi](https://uffizzi.com/) previews. 
Disclaimer: I work on Uffizzi
Uffizzi is a Open Source full stack previews engine and is free for Redash. This will provide newcomers with previews of their PRs in the cloud, allowing them iterate faster, without having to first set up a complete development environment themselves.

TODO:

- [ ] Intial POC

**Comment by arikfr** [comment]
We were using Netlify for something similar, but it was limited to frontend only changes. Wouldn't mind a full stack environment. 

What would be the limitations? Is there another OSS project currently using Uffizzi?

**Comment by eradman** [comment]
No reply. Closing
